marimbabot_planning/src/planning.cpp

```diff
@@ -85,17 +85,17 @@ moveit::planning_interface::MoveGroupInterface::Plan Planning::plan_to_mallet_po
     tf2::Vector3 goal_position(
         goal_point.point.x,
         goal_point.point.y,
-        goal_point.point.z);
+        goal_point.point.z + 0.1);
 
     // Use bio_ik to solve the inverse kinematics at the goal point
     bio_ik::BioIKKinematicsQueryOptions ik_options;
     ik_options.replace = true;
-    ik_options.return_approximate_solution = false; // Activate for debugging if you get an error 
+    ik_options.return_approximate_solution = true; // Activate for debugging if you get an error 
```

Comment:
Do you want to change back to false it before merging?

Reply:
no harm to keep it for now i think